content/en/downloads/_index.md

```diff
@@ -9,14 +9,36 @@ images: []
 weight: 100
 layout: default
 ---
-The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
+The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [Maven Central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
 
 
 | | Version | Date | License |
 | ---------------------------------------- | ----------- | ----------------- | ---------------- |
 | [Narayana Source Code](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final) | 7.2.2.Final | 2025-05-12 | Apache-2.0
 
-[Legacy Narayana versions](../downloads-other/index.html)
+### GPG Public Key
+
+Starting with version 7.2.3.Final, all Narayana [Maven artifacts](https://mvnrepository.com/artifact/org.jboss.narayana) are signed with JBossTM GPG key.
```

Comment:
Initially I though I would prefer just `JBossTM` in lower case (like in the first commit). Not only does it possibly read with JBoss "Trademark" but rather the key itself looks to be lower case but now I think I might even prefer to request a bigger change and say it is `signed with "jbosstm (maven-gpg-plugin) <narayana-users@googlegroups.com>" GPG key` - what do you think, does that add even more confidence somehow?

In summary, I think it should be either `jbosstm` or `jbosstm (maven-gpg-plugin) <narayana-users@googlegroups.com>" GPG key` and not `JBossTM`

Reply:
I would prefer `jbosstm`. The other version is a bit too long